lib/components/dogstatsd.js

```diff
@@ -41,7 +40,9 @@ function Component (app, opts) {
 
   handlerMonitor = filters.handlerMonitor(client)
   rpcMonitor = filters.rpcMonitor(client)
+  rpcErrorHandler = handlers.rpcErrorHandler(app, client)
 
   app.filter(handlerMonitor)
   app.rpcFilter(rpcMonitor)
+  app.set('rpcErrorHandler', rpcErrorHandler)
```

Comment:
I'm not sure I understand in which scenario this handler will be called. Could you explain it, please?